docs/openstack/edpm_adoption.md

```diff
@@ -152,8 +152,8 @@ done
               #
               # These vars are for the network config templates themselves and are
               # considered EDPM network defaults.
-              neutron_physical_bridge_name: br-ctlplane
-              neutron_public_interface_name: eth1
+              edpm_network_config_bridge_name: br-ctlplane
+              edpm_network_config_interface_name: eth1
```

Comment:
As i see the changes haven't been merged in EDPM yet [1][2] so we should hold off merging these for now.

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/d5132138cb07661f2658c6e86a7864dd2d9ede93/config/samples/dataplane_v1beta1_openstackdataplane.yaml#L80-L81
[2] https://github.com/openstack-k8s-operators/edpm-ansible/pull/201